extensions/guacamole-auth-cas/src/main/java/org/apache/guacamole/auth/cas/user/CASAuthenticatedUser.java

```diff
@@ -50,6 +51,12 @@
      */
     private Map<String, String> tokens;
 
+    /**
+     * The unique identifiers of all user groups which affect the permissions
+     * available to this user.
```

Comment:
I'm not sure that this should reference permissions, mainly because the CAS module does not implement connections or permissions at all, so it may be a little misleading to start referencing it within the context of this module.  While we do fully expect that these groups will be used to assign permissions in other modules (JDBC), within this module it's simply the groups the user is a member of.

Reply:
Please review my changes to see if they're more on the mark....